src/globus_sdk/scopes/data/__init__.py

```diff
@@ -0,0 +1,20 @@
+from .auth import AuthScopes
+from .flows import FlowsScopes, SpecificFlowScopeBuilder
+from .gcs import GCSCollectionScopeBuilder, GCSEndpointScopeBuilder
+from .groups import GroupsScopes, NexusScopes
+from .search import SearchScopes
+from .timer import TimerScopes
```

Comment:
I'm betting this is upset now.

```suggestion
from .timers import TimerScopes
```

Reply:
Oh, it is. I'm sure it is; good catch.